user-manual/transfer/clean-up-names.rst

```diff
@@ -0,0 +1,93 @@
+.. _clean-up-names:
+
+==============
+Clean up names
+==============
+
+The clean up names microservice runs twice during processing - once on the
+Transfer tab and once on the Ingest tab. The purpose of this microservice is to
```

Comment:
Sort of a general style guide question -- do we usually say "Transfer tab" and "Ingest tab" or "during Transfer" or some other archivally-suited way? I think how you are saying it is fine, just wondering about alternatives I've used in the past and what we generally should aim for when describing digipres software!

Reply:
I usually say "on the Transfer tab" or "on the Ingest tab" for clarity, especially when it's a job/task that appears on both. But I think other alternatives are fine, as long as it's obvious where we're all supposed to be looking!